.gitlab/pnnl/README.md

```diff
@@ -0,0 +1,86 @@
+# How to connect GitHub to a PNNL GitLab (push) Mirror
+
+To run on our HPC clusters at PNNL while hosting our code base on GitHub, we utilize PNNL's CI/CD services on GitLab. 
+
+With GitLab premium, integration between GitHub and GitLab is included. This is how we have architected a solution with the base level GitLab offering. 
+
+The PNNL GitLab repository is a push mirror of the GitHub. So whenever a commit is pushed to a pull request or the `main` branch - the changes are pushed to the GitLab and a CI pipeline is triggered.
+
+## Steps in linking the GitLab and GitHub
+
+1. Push mirror & pipeline trigger
+
+In `pnnl_mirror.yaml`, we use the variables `GIT_USER`, `GIT_PASSWORD`, and `PNNL_PIPELINE_TRIGGER`. `GIT_PASSWORD` and `PNNL_PIPELINE_TRIGGER` are generated in GitLab and then added to GitHub.
+
+(TODO: is GIT_USER just something you add directly into github?)
```

Comment:
```suggestion
`GIT_USER` is the username of whoever will be authenticated when pushing to GitLab for the mirror action. Since we then manually trigger CI after that, and explicitly skip CI here, this name is cosmetic.
```